dockerfiles/Dockerfile.ubuntu

```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+# ARG SKOSMOS_TARGZ_RELEASE_URL=https://github.com/NatLibFi/Skosmos/archive/v2.4.tar.gz
+
+# git is necessary for some composer packages e.g. davidstutz/bootstrap-multiselect
+RUN apt-get update && apt-get install -y \
+    apache2 \
+    curl \
+    gettext \
+    git \
+    libapache2-mod-php7.2 \
+    locales \
+    php7.2 \
+    php7.2-curl \
+    php7.2-xsl \
+    php7.2-intl \
+    php7.2-mbstring \
+    php-apcu \
+    php-gettext \
+    php-zip \
+    unzip \
+ && rm -rf /var/lib/apt/lists/*
+
+# https://stackoverflow.com/a/28406007
+# fixes warnings like perl: warning: Setting locale failed.
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    locale-gen \
+        ar_AE.UTF-8 \
+        de_DE.UTF-8 \
+        en_GB.utf8 \
+        en_US.utf8 \
+        es_ES.utf8 \
+        fa_IR \
+        fi_FI.utf8 \
+        fr_FR.utf8 \
+        it_IT.utf8 \
+        nb_NO.utf8 \
+        nl_NL.utf8 \
+        nn_NO.utf8 \
+        pl_PL.utf8 \
+        ru_RU.utf8 \
+        sv_SE.utf8 \
+        zh_CN.utf8
+ENV LANGUAGE=en_US:en  
+ENV LC_ALL=en_US.UTF-8 
+ENV LANG=en_US.UTF-8  
+
+# timezone
+RUN sed -i 's/;date.timezone =/date.timezone = "UTC"/g' /etc/php/7.2/apache2/php.ini
+
+COPY dockerfiles/config/000-default.conf /etc/apache2/sites-available/000-default.conf
+
+RUN a2enmod rewrite
+RUN a2enmod expires
+
+RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+
+# Install Skosmos
```

Comment:
I don't think we want to install Skosmos from a tarball here, even commented out... Instead it would perhaps be better to show how the Skosmos code can be copied from the repository itself? Though the docker-compose file solves this in another way, by mounting a volume (if I understood it correctly). In my opinion, it would perhaps be best to copy the Skosmos files inside the container, so that the container becomes independent of the host system after it's been created.

Reply:
Installing inside the container from git now. I used the latest tag @osma . WDYT? (BTW the latest tag doesn't follow the pattern of the previous ones... `v2.8` vs `v.2.9`)

Reply:
Good catch re: the tag name! Must have been a mistake during the release process. I just pushed a tag `v2.9` to try to fix this after the fact, but I guess it's best to leave the invalid tag there as removing it might break things.

Not sure I like the idea of installing the code from git. I think it would be better to copy the code from the current repo itself, in whatever state it is. This way if you want to try out a change, you can adjust the code and test it in a container, without even having to do a commit, never mind pushing it to the main Skosmos repo (and getting it merged etc).

Reply:
Ah, that's even easier! Will change it today 👍